thelastinuit/22-aug-do-i-get-a-bonus.exs

```diff
@@ -0,0 +1,20 @@
+# https://www.codewars.com/kata/56f6ad906b88de513f000d96
+
+defmodule Codewars.Perks do
+  defstruct bonus: false
+
+  defmacro is_hard_worker(bonus) do
+    quote do: unquote(bonus) == true
+  end
+
+  defmacro is_slacker(bonus) do
+    quote do: unquote(bonus) == false
+  end
+end
+
+defmodule Codewars.Reward do
+  import Codewars.Perks
+
+  def bonus_time(salary, bonus) when is_hard_worker(bonus), do: "$#{salary * 10}"
```

Comment:
this could be far more simple but I wanted to learn guards.
```elixir
def bonus_time(salary, false), do: "$#{salary}"
def bonus_time(salary, true), do: "$#{salary * 10}"
```